Database/schema.sql

```diff
@@ -0,0 +1,29 @@
+DROP TABLE IF EXISTS price_readings, subscriptions, users, products;
+
+CREATE TABLE products(
+    product_id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    url TEXT NOT NULL,
+    product_code TEXT NOT NULL,
+    product_name TEXT NOT NULL,
+    CONSTRAINT product_url_unique UNIQUE (url)
+);
+
+CREATE TABLE users (
+    user_id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    email TEXT NOT NULL,
+    CONSTRAINT user_email_unique UNIQUE (email)
```

Comment:
Is this the same as having UNIQUE at the end of the variable?

Reply:
Yes, thanks this is now moved to the variable